plugins/audio-s5gen2/fu-audio-s5gen2-device.c

```diff
@@ -47,88 +48,96 @@ fu_qc_s5gen2_device_msg_out(FuQcS5gen2Device *self, guint8 *data, gsize data_len
 }
 
 static gboolean
-fu_qc_s5gen2_device_msg_in(FuQcS5gen2Device *self, guint8 *data_in, gsize data_len, GError **error)
+fu_qc_s5gen2_device_msg_in(FuQcS5gen2Device *self,
+			   guint8 *data_in,
+			   gsize data_len,
```

Comment:
`data_insz` would be more normal. or maybe just `buf` and `bufsz` to match other places.